content/en/getting_started/synthetics/api_test.md

```diff
@@ -37,22 +40,25 @@ The example below demonstrates the creation of an [HTTP test][2], a subtype of [
     - Add the URL of the endpoint you want to monitor. If you don’t know what to start with, you can use `https://www.shopist.io/`, a test web application. Defining the endpoint to test automatically populates the name of your test to `Test on www.shopist.io`. You can change your test name to something else if you want to.
     - You can select **Advanced Options** to use custom request headers, authentication credentials, body content, or cookies.
     - You can set tags such as `env:prod` and `app:shopist` on your test. Tags allow you to keep your test suite organized and quickly find tests you're interested in on the homepage.
+    - You can create global variables and locally defined variables in your test options. If you need to use credentials or want to inject a dynamically defined timestamp in your API call, click **Create Local Variable**.
 
 6. Click **Test URL** to trigger a sample test run.
 
-{{< img src="getting_started/synthetics/api-test-config-2.jpg" alt="API test configuration"  style="width:60%;">}}
+{{< img src="getting_started/synthetics/api-test-config-3.png" alt="API test configuration"  style="width:100%;">}}
 
 ### Define assertions
 
 Clicking **Test URL** automatically populates basic assertions about your endpoint's response. Assertions define what a successful test run is.
 
 In this example, three default assertions populate after triggering the sample test run:
 
-{{< img src="getting_started/synthetics/assertions-example.png" alt="Browser test failure"  style="width:90%;">}}
+{{< img src="getting_started/synthetics/assertions-example-2.png" alt="Default assertions"  style="width:100%;">}}
 
 Assertions are fully customizable. To add a custom assertion, click on elements of the response preview or click **New Assertion**. 
 
-{{< img src="getting_started/synthetics/api-test-configuration-2.gif" alt="Browser test failure"  style="width:90%;">}}
+{{< img src="getting_started/synthetics/api-test-configuration.mp4" alt="Example API test configuration"  video="true"  >}}
```

Comment:
Wow didn't know we could add videos in the doc, that's so much more convenient to look at!
For that one, ideally we'd show creating an assertion by clicking on a different header than the one we already have a default assertion on (e.g., the `server` one) and manually add one on the body (e.g., body contains shop.ist). To end up in this state: https://a.cl.ly/bLukvBdl. 
I'm thinking we could also make the video shorter by starting after having clicked on the `Test URL` button since we go over default assertions with the above screenshot.